Assets/Scripts/Editor/EditorSceneLoader.cs

```diff
@@ -0,0 +1,29 @@
+using UnityEditor;
+using UnityEditor.SceneManagement;
+
+namespace Celeritas.Editor
+{
+	public class EditorSceneLoader
```

Comment:
Same comment as below, would be nice to have a /// comment describing the responsibilities of this class. Not mandatory, but would be nice.